extensions/broker-server-postgres-flyway-jooq/build.gradle.kts

```diff
@@ -22,37 +22,37 @@ val postgresVersion: String by project
 
 buildscript {
     dependencies {
-        classpath("org.testcontainers:postgresql:1.19.1")
+        classpath(libs.testcontainers.postgresql)
     }
 }
 
 plugins {
-    id("org.flywaydb.flyway") version "9.21.1"
-    id("nu.studer.jooq") version "7.1.1"
+    alias(libs.plugins.flyway)
+    alias(libs.plugins.jooq)
     `java-library`
     `maven-publish`
 }
 
 dependencies {
-    api("org.jooq:jooq:3.18.7")
-    api("com.github.t9t.jooq:jooq-postgresql-json:4.0.0")
+    api(libs.jooq.jooq)
+    api(libs.t9tJooq)
```

Comment:
library should be called "jooqPostgresJsonExt"

Reply:
Ok, but it will be under `t9tJooq` so no need to indicate `...Ext`